internal/errcheck/errcheck.go

```diff
@@ -222,14 +224,25 @@ type visitor struct {
 	errors []UncheckedError
 }
 
-func (v *visitor) fullName(call *ast.CallExpr) (string, bool) {
+func (v *visitor) selectorAndFunc(call *ast.CallExpr) (*ast.SelectorExpr, *types.Func, bool) {
```

Comment:
While you're here, can you add a comment to this function, particularly the return values?

Reply:
:+1: